Jenkins/buildscript.py

```diff
@@ -34,6 +34,11 @@
 
 ################### Functions #########################################################
 
+def make_scl_command(command_list):
+    command_str = " ".join(command_list)
+    quoted_command_str = "\"{0}\"".format(command_str)
+    return " ".join(["scl", "enable", "devtoolset-2", "{0}".format(quoted_command_str)])
```

Comment:
@martyngigg It's just occurred to me that this will need to be in place for the new starters Monday. I don't think I'm going to get a chance to do the RHEL6 check as I'm on holiday Thursday and Friday.